### PR TITLE
People can be cloned from heads or brains

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
balance: Nanotrasen Cloning Divison's three hundred identical scientists
have announced an upgrade to the cloning computer's software. You can
now scan brains or dismembered heads and successfully clone from them.
Brains seem to remember the DNA of the last body they were attached to.
/:cl:

Both brains and heads can be cloned. Codewise, brainmobs already
remember their old mob's DNA. Existing requirements of being "in body"
still apply.

It makes little in universe sense why it is required for a head to be
attached to a torso before you can scan it, or why a brain needs to be
inside a head attached to a torso to be valid to scan.

This _will_ make it easier for medbay to clone people who have been
reduced to organs.

### CODE REVIEW

**Comment by jughu** [comment]
Wasnt the whole point of dismemberment not being cloned easily

**Comment by Aloraydrel** [comment]
It was a side effect of it

File: code/game/machinery/computer/cloning.dm [hunk 1]
```diff
@@ -435,64 +433,70 @@
...
+	if(!istype(dna))
 		scantemp = "<font class='bad'>Unable to locate valid genetic data.</font>"
 		playsound(src, 'sound/machines/terminal_prompt_deny.ogg', 50, 0)
 		return
-	if (!subject.getorgan(/obj/item/organ/brain))
-		scantemp = "<font class='bad'>No signs of intelligence detected.</font>"
-		playsound(src, 'sound/machines/terminal_alert.ogg', 50, 0)
-		return
-	if (subject.suiciding == 1 || subject.hellbound)
+	if(mob_occupant.suiciding == 1 || mob_occupant.hellbound)
```
Comment (RemieRichards) [inline code/game/machinery/computer/cloning.dm]: ...

**Comment by CPTANT** [comment]
Yeah this was super annoying and inconsistent. 

Either the body or the head needs to be clonable and now neither is. Since the cloner checks for minds the head is the logical choice.

**Comment by FantasticFwoosh** [comment]
Design lead @KorPhaeron  didn't express support for it over IRC and its a annoying powercreep because heads can be reattached to bodies (any body at all) with surgery in a few simple steps. This is lazy and counter productive to the bare minimum requirement to put a head on some shoulders even when it is made as simple as possible.

> Come back when you've actually re-constructed a entire human being out of nothing but a brain & a leg salvaged from a explosion and revived them with cloning or strange reagent frankenstiening.

**Comment by Haevacht** [comment]
This is down to a Kor vote, probably but I stand against this shit.

**Comment by BeeSting12** [comment]
no, take thirty seconds to reattach it to a body it isnt that hard.

**Comment by Cyberboss** [comment]
better yet, do brain surgery on a live monkey man and then defib them

**Comment by SgtxRootbeer** [comment]
Well, at least I get to keep my snowflake character after it's been husked

**Comment by XDTM** [comment]
Why would you need a body, it doesn't make sense to require it. And dismemberment made it so killers could make people unclonable with no effort, at least now you'd have to pick up the head and hide or burn it, instead of leaving it around knowing nobody will spend time getting genetics and surgery access to get a monkeyman to graft the head onto.

**Comment by KorPhaeron** [comment]
@coiax 

Couldn't you bypass the husk thing pretty easily like this

**Comment by Iamgoofball** [comment]
@coiax 
Can you lock this to t3 or t4 parts at least? Add a small timegate.

**Comment by Iamgoofball** [comment]
@SgtxRootbeer 
>revealing the secret monkey technique

**Comment by RemieRichards** [comment]
@KorPhaeron just make husking husk the organs too.

**Comment by Repukan** [comment]
Hugbox

**Comment by coiax** [comment]
Yeah, husking the organs would be cool.

**Comment by FantasticFwoosh** [comment]
Way to make surgery even more pointless, how are you even meant to revitalise organs if they now have comprehensive health systems to compare them between (healthy = husked)

Id assume it'd be by slapping on synthflesh or printing off another one but that's really sloppy.

**Comment by RemieRichards** [comment]
@FantasticFwoosh less sloppier than "organs are nonsense objects that don't react to anything happening to them including what is ICly all of the blood being sucked out of a person"
That would damage the organs, there's no need for them to be these invulnerable objects, it's dumb and unfun.

**Comment by FantasticFwoosh** [comment]
If you make organs take damage there really will be nothing left of a person after a bomb explosion to scoop up, making this change for scooping up organs **"easier"** irrelevant.

**Comment by Iamgoofball** [comment]
OOOOORRRR

we add synthlimb/synthbrain~~/synthbutt~~

**Comment by FantasticFwoosh** [comment]
Synthbrain would probably result in a faux ling brain and or ckeyless monkeyman brain.

**Comment by XDTM** [comment]
what is the point of a synthbrain if the soul is in the old brain

**Comment by FantasticFwoosh** [comment]
Oh great your brain is husked and on fire, time to permanently delete your brain by throwing it into a bonfire/lava

**Comment by jughu** [comment]
Synth brain burgers

**Comment by Tacolizard** [comment]
A major part of robotics is being able to revive people who have lost their body. This is a big nerf to that.

**Comment by Core0verload** [comment]
I'm fine with this if it's gated behind T3 or T4. Other than that, no.

**Comment by bgobandit** [comment]
I'm torn

like there's no logical, IC, reason why you shouldn't be able to clone someone from a head or a brain, especially when you can do it with blood (podmen)

but this will massively affect game balance

**Comment by PKPenguin321** [comment]
>but this will massively affect game balance

no, if you have the head/brain already you can stitch it onto literally any corpse/monkeyman to clone it already. this just gives doctors less to do for "QoL"
👎

**Comment by bgobandit** [comment]
that's literally how cloning (the actual cloning) came about though

**Comment by RemieRichards** [comment]
@Core0verload which is why I said to husk the organs, as husking can only be cloned at the higher (highest?) tier.

**Comment by factoryman942** [comment]
Make the cloner only make talking heads

**Comment by FantasticFwoosh** [comment]
@RemieRichards in doing that outside of a actual self made PR you're kind of undermining the premise that organs would be easy to recover as @coiax wants from the OP by summarily making them easy to husk by certain means when on-topic this discussion is about whether to clone from abstract limbs.

> Which initially wont work, since the head with the brain is the only body part to actually carry any id unless a random cut off leg could somehow backtrace its .owner for genetic code.

**Comment by RemieRichards** [comment]
@FantasticFwoosh It's on topic as people feel it's not balanced at the moment, huskable limbs would balance that, that should be obvious to see.

**Comment by InsaneHyena** [comment]
No. This is what robotics are for. Stop stealing our jobs.

**Comment by FantasticFwoosh** [comment]
It can have its own discussion but right now we're discussing whether heads should be suitable for cloning ckeys subsitute for chests with a head attached. All the limb husking is made irrelevant by cybernetic limbs that can just be mechanically repaired with wires & welders anyway.

> How do you think i manage to revive people without bodies, without also riddling the body with toxicity due to it not belonging to the owner? Usually i just get a donated corpse, fully aug it, slap a brain in and strange reagent it or piece together a workable body.

**Comment by Tacolizard** [comment]
With this merged people will have even less of a reason to get borged.

**Comment by PKPenguin321** [comment]
>that's literally how cloning (the actual cloning) came about though

No, that was making a bug into a feature, this is obsoleting a feature (head/brain swapping) with a "QoL" feature

**Comment by lzimann** [comment]
@KorPhaeron say yes or no

**Comment by FantasticFwoosh** [comment]
So what's it gonna be @KorPhaeron 



**Or**

**Comment by XDTM** [comment]
i'm still wondering why having to stitch a head to a stump to scan it and trash it is considered particularly good gameplay, it's just a code remnant due to the torso being the mob

surgery is fun but not when the body you work on isn't even used

**Comment by KorPhaeron** [comment]
Robotics was used for like 6 years on /tg/ before dismemberment was even possible so its very odd for people to be saying this is some sort of critical balance thing that will kill the department

**Comment by PKPenguin321** [comment]
i didnt mention robotics i mentioned making docs have to stitch the gosh dang head back on before cloning which i actually enjoy doing as doctor it's gratifying

**Comment by FantasticFwoosh** [comment]
Well really it doesn't even have to be the head it purportedly belongs to @PKPenguin321 but thats easy to digress. Doctors are already not hands on enough with re-attachment because the cloner is really just too cheap

> The really good doctors shine through when they on-purposefully eject a clone (safely) early out of a cloner, and fix them up with additional limbs/organs and such so the next person can be speedcloned when there is a VIP like a head of staff waiting in queue.

**Comment by XDTM** [comment]
If you want more surgery add more useful surgical procedures, and if you think cloning is too efficient nerf the cloning time, don't gate it with a mechanic that was only there due to code limitations and makes no sense otherwise.

**Comment by PKPenguin321** [comment]
No you can't just add more surgeries, they won't be done 99% of the time and aren't really gratifying unless they're required to raise the dead, which stitching heads on currently is

**Comment by Scinexx** [comment]
I can agree heavily with bgobandit that consistency/IC wise it doesn't make sense to not be able to clone, though there are balancing issues.

I'm just spitballing here, but what about the character receives a penalty once cloned if the body is gone, like them coming out of the pod injured, or in crit? Possibly even genetic defects requiring mutadone. You could justify this because you could assume the brain went through trauma or sustained at least some damage.

**Comment by XDTM** [comment]
@Pkpenguin321 you can still brainswap the target to a monkeyman and defib to completely skip cloning, which imo feels better thsn just fulfilling an arbitrary cloning requirement

**Comment by lzimann** [comment]
@tgstation/commit-access so?

**Comment by optimumtact** [comment]
It's a no from me :-1: because I like the idea of transplanting their head onto a monkey to clone it

**Comment by RemieRichards** [comment]
It's a 👍 from me.

**Comment by AnturK** [comment]
I'm okay with this.

**Comment by ohnopigeons** [comment]
There were pretty vocal complaints about this on both Sybil and Basil.

**Comment by CPTANT** [comment]
What were the arguments given?

**Comment by PKPenguin321** [comment]
delete this!!!!!!

**Pull request merged**